AI-and-Analytics/Getting-Started-Samples/INC-Quantization-Sample-for-PyTorch/README.MD

```diff
@@ -104,37 +93,20 @@ The sample contains one Jupyter Notebook and one Python script.
 
```

Comment:
Change title case of Run the Notebook to ### Run the Notebook

Reply:
addressed it accordingly